src/util/ssl_utils.js

```diff
@@ -46,7 +46,8 @@ const certs = {
     S3: new CertInfo(config.S3_SERVICE_CERT_PATH),
     EXTERNAL_DB: new CertInfo(config.EXTERNAL_DB_SERVICE_CERT_PATH),
     STS: new CertInfo(config.STS_SERVICE_CERT_PATH),
-    IAM: new CertInfo(config.IAM_SERVICE_CERT_PATH)
+    IAM: new CertInfo(config.IAM_SERVICE_CERT_PATH),
+    METRICS: new CertInfo(config.S3_SERVICE_CERT_PATH) // metric server will use the S3 cert.
```

Comment:
I know you have been told in the design meeting to use the S3 cert but I know @Neon-White had some work on changing STS cert to not re-use the S3 cert due to an issue mentioned in https://github.com/noobaa/noobaa-core/pull/8123
so per what Ben wrote, we might have an issue with this re-use on containerized env, Ben WDYT?
CC: @nimrod-becker

Reply:
on the other hand, on non containerized - we currently have the certificates under certificates/ dir, and adding a new certificate will require some changes in the code/structure for allowing different certificates for different services.

Reply:
yep, thanks Ben! so I looks like we can re-use the S3-cert, but this must be documented

Reply:
Sure will update the doc